.github/workflows/release.yaml

```diff
@@ -150,3 +175,20 @@ jobs:
           docker run --rm redocly/cli lint https://raw.githubusercontent.com/Rebilly/api-definitions/main/openapi/openapi.yaml
           docker run --rm redocly/cli bundle https://raw.githubusercontent.com/Rebilly/api-definitions/main/openapi/openapi.yaml
           docker run --rm redocly/cli build-docs https://raw.githubusercontent.com/Rebilly/api-definitions/main/openapi/openapi.yaml
+
+  notify:
+    needs:
+      - release
+      - post-release-smoke-checks
+    if: needs.release.outputs.published == 'true'
+    name: Post the Release Message
+    runs-on: ubuntu-latest
+    steps:
+      - name: Post to a Slack channel
+        id: slack
+        uses: slackapi/slack-github-action@v1.23.0
```

Comment:
We may need to add a small sleep but maybe not relevant here.

Reply:
I'm going to check if this works. If not, I'll add the timeout next time.